### ISSUE TITLE
Websocket error that prevents new connections to the database

### ISSUE THREAD

**Issue by kim** [clockworklabs/SpacetimeDB#224]
sometimes when publishing (with clear fwiw) you get this error:

```rust
WARN crates/client-api/src/routes/subscribe.rs:177: error closing: WebSocket protocol error: Sending after closing is not allowed
```

and after that no new connection (including logs) can be made until you clear and re-publish

update: this seems to happen when you publish with clear flag while there's a connected client (like editor window that we always have running)

**Comment by kim** [comment]
I think the problem occurs when a module is updated -- the old subscription becomes invalid because the old module got torn down.

**Comment by kim** [comment]
https://github.com/clockworklabs/SpacetimeDB/blob/master/crates/client-api/src/routes/subscribe.rs#L51-L52

**Comment by coolreader18** [comment]
@kim can you verify that this still exists? I think Phoebe might've fixed this with #235

**Comment by coolreader18** [comment]
Or well, fixed it for the rust quickstart-chat client
